app/services/notices/setup/abstraction-alerts/fetch-monitoring-station.service.js

```diff
@@ -0,0 +1,110 @@
+'use strict'
+
+/**
+ * Orchestrates fetching the data needed for the Monitoring station journey
+ * @module FetchMonitoringStationService
+ */
+
+const { db } = require('../../../../../db/db.js')
+
+/**
+ * Orchestrates fetching the data needed for the Monitoring station journey
+ *
+ * @param {string} id
+ * @returns {Promise<{object}>}
```

Comment:
I think this should be

```suggestion
 * @returns {Promise<object[]>}
```